_src/whitepaper/index.md

```diff
@@ -1,23 +1,27 @@
 ---
 layout: page
 
-title: Whitepaper
-description: "We're working on a new whitepaper. Stay tuned for updates."
+title: BigchainDB 2.0 Whitepaper
+description: >
+    BigchainDB version 2.0 makes significant improvements over previous versions. In particular, it is now Byzantine fault tolerant (BFT), so up to a third of the nodes can fail in any way, and the system will continue to agree on how to proceed. BigchainDB 2.0 is also production-ready for many use cases.
 
 image: photo3.jpg
 header: photo3.jpg
 
 whitepaper:
-    title: Whitepaper downloads
-    text: The whitepaper is no longer a living document. Significant changes made since June 8, 2016 are noted in an Addendum attached at the end.
-    updated: 2017-05-23
+    title: Whitepaper Downloads
+    text: The whitepaper is a living document...
```

Comment:
I'm not sure we need this line. It reminds me of those "under construction" disclaimers that used to be on all websites in the 1990s.

Reply:
![anim0205-1_e0](https://user-images.githubusercontent.com/90316/40109930-ef086372-58fe-11e8-8f71-745d5ca49341.gif)